app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +59,8 @@ def summary
 
   private
 
-  def devise_job_alert_search_criteria
-    @devised_job_alert_search_criteria = Search::CriteriaDeviser.new(vacancy).criteria
+  def invent_job_alert_search_criteria
+    @invented_job_alert_search_criteria = Search::CriteriaInventor.new(vacancy).criteria
```

Comment:
This method sets `@invented_job_alert_search_criteria`, right? Why is this being called before the show action in a callback when it's set in the show action anyway?